testsuite/MDAnalysisTests/lib/test_distances.py

```diff
@@ -20,7 +20,6 @@
 # MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
-from tkinter import FALSE
```

Comment:
@hmacdope git blame identifies this as an addition from one of your PRs - I can't work out why it's there, is there any reason for it? (not having it fixes this issue)

Reply:
I have no idea where on earth this came from. Thanks for removing. :)